client/src/components/userpage.css

```diff
@@ -0,0 +1,15 @@
+.selector::first-letter {
+  color: #f09600;
+}
+
+.selector {
+  text-align: center;
+  align-content: center;
+  margin: auto;
+}
+
+html,
+body {
+  max-width: 100%;
+  overflow-x: hidden;
+}
```

Comment:
this file should be unnecessary. We don't need css outside of the root html page